samples/apps/copilot-chat-app/webapi/Config/AIServiceOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace SemanticKernel.Service.Config;
+
+/// <summary>
+/// Configuration options for AI services, such as Azure OpenAI and OpenAI.
+/// </summary>
+public sealed class AIServiceOptions
+{
+    public const string CompletionPropertyName = "Completion";
+    public const string EmbeddingPropertyName = "Embedding";
+
+    public enum AIServiceType
```

Comment:
Do what you want with this comment, but I prefer to only use strings instead of relying on enums whenever those enum values are exposed out of our code (Use "AzureOpenAI" and "OpenAI" instead of 0 and 1).

Reply:
enums are safer and faster than strings. 

Also, whenever a class changes behavior using an enum, consider having 2 classes instead. I don't think this is the case, just sharing my experience.

Reply:
As this is a small change to the current pattern, I will leave this as-is for this PR and we will discuss more this week.